src/Microsoft.Extensions.ApiDescription.Client/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -174,30 +176,36 @@
   <!-- DefaultDocumentGenerator -->
 
   <Target Name="DefaultDocumentGenerator">
-    <Message Importance="high" Text="%0ADefaultDocumentGenerator:" />
-    <Message Importance="high" Text="  @(CurrentServiceProjectReference): %(DocumentPath)" />
-
     <ItemGroup>
       <!-- @(CurrentServiceProjectReference) item group will never contain more than one item. -->
       <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
         <Command>dotnet getdocument --no-build --project %(FullPath) --output %(DocumentPath)</Command>
         <DefaultDocumentGeneratorOptions
             Condition="'%(DefaultDocumentGeneratorOptions)' == ''">$(DefaultDocumentGeneratorDefaultOptions)</DefaultDocumentGeneratorOptions>
+        <ProjectConfiguration
+            Condition="'%(ProjectConfiguration)' == ''">$(Configuration)</ProjectConfiguration>
       </CurrentServiceProjectReference>
-      <CurrentServiceProjectReference Update="@(_Temporary)">
+      <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
         <Command>%(Command) --framework %(ProjectTargetFramework)</Command>
-        <Command Condition="'%(ProjectConfiguration)' == ''">%(Command) --configuration $(Configuration)</Command>
-        <Command Condition="'%(ProjectConfiguration)' != ''">%(Command) --configuration %(ProjectConfiguration)</Command>
+      </CurrentServiceProjectReference>
+      <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
         <Command Condition="'%(Method)' != ''">%(Command) --method %(Method)</Command>
+      </CurrentServiceProjectReference>
+      <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
         <Command Condition="'%(Service)' != ''">%(Command) --service %(Service)</Command>
+      </CurrentServiceProjectReference>
+      <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
         <Command Condition="'%(Uri)' != ''">%(Command) --uri %(Uri)</Command>
-        <Command
-            Condition="'%(DefaultDocumentGeneratorOptions)' != ''">%(Command) %(DefaultDocumentGeneratorOptions)</Command>
+      </CurrentServiceProjectReference>
+      <CurrentServiceProjectReference Update="@(CurrentServiceProjectReference)">
+        <Command>%(Command) --configuration %(ProjectConfiguration) %(DefaultDocumentGeneratorOptions)</Command>
       </CurrentServiceProjectReference>
     </ItemGroup>
 
-    <Message Importance="high" Text="%0A%(CurrentServiceProjectReference.Command)" />
-    <Exec IgnoreExitCode="$([System.IO.File]::Exists('%(DocumentPath)'))" Command="%(CurrentServiceProjectReference.Command)" />
+    <Message Importance="high" Text="%0ADefaultDocumentGenerator:" />
+    <Message Importance="high" Text="  %(CurrentServiceProjectReference.Command)" />
+    <Exec Command="%(CurrentServiceProjectReference.Command)"
+        IgnoreExitCode="$([System.IO.File]::Exists('%(DocumentPath)'))" />
```

Comment:
You could use the `EXISTS` function here

Reply:
This suggestion didn't work. `EXISTS` only works in a `Condition`.